.github/workflows/test.yml

```diff
@@ -8,6 +8,8 @@ jobs:
       - uses: actions/checkout@v2
       - run: yarn install --frozen-lockfile --ignore-scripts
       - run: yarn build
+      # Ensures that building did not modify the working directory
+      - run: git diff --quiet
```

Comment:
Suggestion: maybe instead of explaining this with a comment, you can add a `|| echo ''' && exit 1` with an error message? That way the actions output might be more comprehensible when it fails.

Reply:
I tried exactly that but it would call `exit 1` anyway! My solution was to move the logic into a script in 1da4655, but I abandoned that for `git diff --quiet` instead. 🤔

Reply:
Ah, TIL the command combination operator (`{}`). Got it working in 07eb100 and it fails as expected in #11.